src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -42,13 +44,22 @@ def read_and_clean_all_sheets(excelfile: str | Path) -> dict[str, pd.DataFrame]:
         # Credits: https://stackoverflow.com/a/70537454/14414188
         with mock.patch("openpyxl.styles.fonts.Font.family.max", new=100):
             df_dict = pd.read_excel(excelfile, sheet_name=None)
+    _find_duplicate_col_names(str(excelfile), list(df_dict))
     try:
         return {name.strip(""): clean_data_frame(df) for name, df in df_dict.items()}
     except AttributeError:
         msg = InvalidSheetNameProblem(str(excelfile), list(df_dict.keys())).execute_error_protocol()
         raise InputError(msg) from None
 
 
+def _find_duplicate_col_names(excelfile: str, col_names: list[str]) -> None:
+    sheet_names = [str(x).lower().strip() for x in col_names]
```

Comment:
For some functions, we strip the names in a dict comprehension without checking if that would result in two identical keys.